Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Theramorean Culture Innovations

## Changelog:
- Made Theramore equivalent to Kul Tiran in terms of innovation, given the culture has split off the latter.

## Tests:

# How to test:
- Play as Jaina in 603 and set sail to Theramore, the culture should diverge and have all innovations from Kul Tiras.
- Play as Jaina in 605 and check if all innovations are present and compare them with Kul Tiras' ones.
